phir/model.py

```diff
@@ -243,14 +261,14 @@ class MOp(Op):
 
 
 QOp: TypeAlias = MeasOp | SQOp | TQOp
-OpType: TypeAlias = FFCall | COp | QOp | MOp
+OpType: TypeAlias = FFCall | COp | QOp | MOp | Barrier
```

Comment:
This is required to be able to treat Barrier as, e.g., an op under a conditional.